docs/nb/bulk_validation_referential_integrity_check.ipynb

```diff
@@ -662,64 +673,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 33,
+   "execution_count": 41,
    "id": "29ec7e82-d079-4525-bd7b-d770fd69d788",
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'_id': ObjectId('66748e9ff33c8467941f01b4'),\n",
-       " 'id': 'nmdc:omprc-13-4wkf0639',\n",
-       " 'name': 'Rachael_21T_04-15A_M_14Mar17_leopard_Infuse',\n",
-       " 'instrument_name': '21T Agilent',\n",
-       " 'has_input': ['nmdc:bsm-13-4bfysc34'],\n",
-       " 'has_output': ['nmdc:dobj-13-xx781m34'],\n",
-       " 'omics_type': {'has_raw_value': 'Organic Matter Characterization'},\n",
-       " 'part_of': ['nmdc:sty-11-33fbta56'],\n",
-       " 'description': 'High resolution MS spectra only',\n",
-       " 'processing_institution': 'EMSL',\n",
-       " 'gold_sequencing_project_identifiers': [],\n",
-       " 'alternative_identifiers': ['emsl:570856'],\n",
+       "{'_id': ObjectId('66a8f648b3ed1b2200050335'),\n",
+       " 'id': 'nmdc:omprc-11-sxze4w22',\n",
+       " 'has_input': ['nmdc:bsm-11-978cs285'],\n",
+       " 'has_output': ['nmdc:dobj-11-1epz0d53'],\n",
+       " 'part_of': ['nmdc:sty-11-28tm5d36'],\n",
        " 'type': ['OmicsProcessing', 'PlannedProcess', 'NamedThing']}"
```

Comment:
OmicsProcessing should be DataGeneration.

Reply:
I created issue https://github.com/microbiomedata/nmdc-runtime/issues/629 about this.